aws/table_aws_cloudformation_stack_set.go

```diff
@@ -0,0 +1,282 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+
+	cloudformationv1 "github.com/aws/aws-sdk-go/service/cloudformation"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudFormationStackSet(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_cloudformation_stack_set",
+		Description: "AWS CloudFormation Stack Set",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("stack_set_name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException", "StackSetNotFoundException"}),
```

Comment:
@ParthaI Do we need to ignore both the error codes?

Reply:
@misraved, Apologies for the confusion. We actually only need to ignore the error code `StackSetNotFoundException`. According to the [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_DescribeStackSet.html), it states that if the resource does not exist, we should receive a `StackSetNotFound` error. However, in our case, we are receiving a `StackSetNotFoundException` error instead.